syntax = "proto3";
option optimize_for = LITE_RUNTIME;

import "apps/libmc/sim/protobuf/system_message.proto";

package libmc.sim;

message SimMessage {
  // Message that handles system actions and statuses.
  SystemMessage system = 1;
}
